.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: go
+env:
+  global:
+    - GO111MODULE: on
```

Comment:
looks like Travis is converting this to `GO111MODULE="true"`, use `GO111MODULE: "on"` instead.

Reply:
That seems to have done the trick, but now the build process fails because it tries to fetch "github.com/mholt/caddy" even though there is no mention of the old repository path anymore.

Reply:
ok, we can try to merge this PR and see if it fixes this issue.

Reply:
You need to publish a new release for Go to pick up these changes, see #41.

Reply:
I see